server/controllers/documentController.js

```diff
@@ -1,155 +1,163 @@
 import { Document } from '../models';
-import errorMessages from '../constants/errors';
-import successMessages from '../constants/successes';
-import helpers from '../helpers/helpers';
+import errorConstants from '../constants/errorConstants';
+import successConstants from '../constants/successConstants';
+import documentHelpers from '../helpers/documentHelpers';
 
-const getPageMetadata = helpers.getPageMetadata;
-export default {
+const getPageMetadata = documentHelpers.getPageMetadata;
+const documentControllers = {
```

Comment:
Best Practices - It's always good to start class names with upper case

Reply:
It isn't a class because. If you check where they are defined, you'd discover that they are object. If I have defined them with a constructor function or a class keyword, then I would be mandated to use uppercase